sunpy/extern/parse.py

```diff
@@ -471,7 +471,7 @@
 
 __all__ = 'parse search findall with_pattern'.split()
 
-log = logging.getLogger(__name__)
+log = logging.getLogger("parse")
```

Comment:
This is annoying, as we now have to remember to change this on every update.

Reply:
For context, not doing this was causing an error to be raised when the log level was set to debug. I assume because the name of this logger started with `sunpy` it was being sent through some of the astropy logging infrastructure in a way that it wasn't supposed to be and throwing a traceback.